recipe/set_bld_opts.bat

```diff
@@ -86,5 +86,11 @@ set BLD_OPTS=%WIN64% ^
     POPPLER_MAJOR_VERSION=%POPPLER_MAJOR_VERSION% ^
     POPPLER_MINOR_VERSION=%POPPLER_MINOR_VERSION% ^
     POPPLER_LIBS="%LIBRARY_LIB%\poppler.lib"
+    WEBP_ENABLED=YES
+    WEBP_CFLAGS="-I%LIBRARY_INC%"
+    WEBP_LIBS="%LIBARY_LIB%\libwebp_a.lib"
```

Comment:
I think it is just `libwebp.lib` ?

BTW I suspect `libwebp` is not compiled for use in a DLL... Looking into.

Reply:
See https://github.com/conda-forge/libwebp-feedstock/pull/20